build.rs

```diff
@@ -1,417 +1,110 @@
-mod build_support;
+use std::{env, path::PathBuf};
 
-use std::{
-    env, fs, io,
-    path::{Path, PathBuf},
-    process::Command,
-    time::SystemTime,
-};
+// Compiler specific compiler flags for CMake
+fn compiler_flags() -> Vec<&'static str> {
+    let mut flags = Vec::new();
 
-use build_support::{static_lib_filename, Target};
-use flate2::{read::GzDecoder, write::GzEncoder, Compression};
-use serde::{Deserialize, Serialize};
+    if cfg!(target_env = "msvc") {
+        flags.push("/EHsc");
```

Comment:
I am not very familiar with windows.
Can you, please, explain me in a few words what does this flag do? :)

Reply:
I had to add it to build on Windows. It specifies which exception handling model to use.

See https://learn.microsoft.com/en-us/cpp/build/reference/eh-exception-handling-model?view=msvc-170